extra/1-radio-stations.js

```diff
@@ -14,10 +14,18 @@
  */
```

Comment:
Near perfect attempt, looks like you rushed a bit and ran into some problems, but they're very small, the gist is correct